build/android/BUILD.gn

```diff
@@ -5,22 +5,30 @@ import("//tools/grit/grit_rule.gni")
 action("sign_app") {
   script = "//brave/build/android/sign_apk.py"
   _apksigner = "$android_sdk_build_tools/apksigner"
+  _jarsigner = "//third_party/jdk/current/bin/jarsigner"
   _zipalign = "$android_sdk_build_tools/zipalign"
 
   deps = [ "//brave/app/android:create_dist_zips" ]
 
-  target_sign_app_path = ""
-  if (target_cpu == "arm64" || target_cpu == "x64" ||
-      target_apk_base == "mono") {
-    target_sign_app_path = "$root_out_dir/apks/MonochromePublic.apk"
+  if (target_cpu == "arm64" || target_cpu == "x64") {
+    target_sign_app_path = "$root_out_dir/apks/MonochromePublic6432.aab"
   } else {
-    target_sign_app_path = "$root_out_dir/apks/ChromePublic.apk"
+    if (target_apk_base == "mono") {
+      #target_sign_app_path = "$root_out_dir/apks/MonochromePublic.apk"
```

Comment:
nit: if this line is left here commented out, it is better to add the comment, why the line is left here; otherwise it is better to remove the commented line

Reply:
I left the lines commented out in case anyone want to build apk instead. As mentioned below I agree it is better to have a flag so I will add a flag to address this.